src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--sun calendar--month-length--29">
```
ось тут повинні бути модифікатори, і коли ти будеш міняти значення, твій календар на сторінці буде мінятися

Reply:
not fixed

Reply:
your modifiers work, just add them!